src/test/java/gov/cabinetoffice/gap/adminbackend/services/GrantExportServiceTest.java

```diff
@@ -72,19 +75,34 @@ void successfullyGetGrantExports() {
                     .id(id)
                     .applicationId(1)
                     .status(GrantExportStatus.COMPLETE)
+                    .created(date)
                     .createdBy(1)
-                    .lastUpdated(Instant.now())
```

Comment:
Is there a reason the date creation was moved out from here? No big deal either way, just curious if it's a unit test quirk

Reply:
i think is just for reusability, as she's using the Instant.now also in the created property

Reply:
The seconds were slightly off with each Instant.now() which was causing the tests to fail